### PR TITLE
Development jan

### PR DESCRIPTION
**Pull request by Japhiolite**
added small snipped to hide legend if no data is plotted on a section and updated the name of `Computational Geology` to `Computational Geosciences and Reservoir Engineering` in the about.rst

### CODE REVIEW

File: docs/source/about.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 About us
 ========
 
-Computational Geology RWTH Aachen University.
+Computational Geosciences and Reservoir Engineering RWTH Aachen University.
```
Comment (Leguark) [inline docs/source/about.rst]: German precision. I am sure Florian will like it

File: gempy/plot/plot.py [hunk 2]
```diff
@@ -217,7 +217,8 @@ def plot_predef_sections(model, show_traces=True, show_data=False, section_names
 
 
 def plot_section(model, cell_number=13, block=None, direction="y", interpolation='none',
-                 show_data=False, show_faults=True, show_topo = False,  block_type=None, ve=1, **kwargs):
+                 show_data=False, show_faults=True, show_topo=False, show_legend=False,
```
Comment (Leguark) [inline gempy/plot/plot.py]: Do you think that by default is better False than True?

Comment (Japhiolite) [reply gempy/plot/plot.py]: hmm, good question! Thinking about it, default `True` might be better, as people will initially want to have it I think..
```suggestion
                 show_data=False, show_faults=True, show_topo=False, show_legend=True, 
```

File: gempy/plot/visualization_2d.py [hunk 3]
```diff
@@ -259,7 +259,7 @@ def plot_map(self, solution: Solution, contour_lines=True):
 
     def plot_block_section(self, solution:Solution, cell_number:int, block:np.ndarray=None, direction:str="y",
                            interpolation:str='none', show_data:bool=False, show_faults:bool=False, show_topo:bool=False,
-                           block_type=None, ve:float=1, **kwargs):
+                           show_legend:bool=False, block_type=None, ve:float=1, **kwargs):
```
Comment (Japhiolite) [inline gempy/plot/visualization_2d.py]:
```suggestion
                           show_legend:bool=True, block_type=None, ve:float=1, **kwargs):
```

**Comment by Leguark** [comment]
Great! Thank you jan

**Pull request merged**